content/GettingStarted/SCALEReleaseNotes.md

```diff
@@ -38,30 +44,77 @@ weight: 10
 |                       | Tag | 18 December 2023 |
 |                       | **Release** | **19 December 2023** |
 {{< /truetable >}}
-## Obtaining a Release
+{{< /expand >}}
+
+## Upgrade Notes
+
+Several built-in services from SCALE 22.12 (Bluefin) or TrueNAS CORE 13.0 in **System Settings > Services** are replaced by community applications.
+You must disable these built-in services and begin using the equivalent application **before** upgrading to SCALE 23.10 (Cobia).
+
+{{< expand "Replaced Services (Click to expand)" "v" >}}
+{{< columns >}}
+* Dynamic DNS replaced by **[ddns-updater]({{< relref "ddns-updater.md" >}})**
+* OpenVPN Server replaced by multiple VPN [apps]({{< relref "/SCALETutorials/Apps/CommunityApps/_index.md" >}})
+* OpenVPN Client has no equivalent application.
+* Rsyncd Server replaced by **[rsyncd]({{< relref "rsyncd.md" >}})**
+* WebDAV replaced by **[webdav]({{< relref "webdav.md" >}})**
+* TFTP replaced by **[tftpd-hpa]({{< relref "tftp-hpaapp.md" >}})**
+<--->
```

Comment:
This rendered looks weird when the content in lie 66 looks like it is missing the bullet point. Can you change this so that line 66 renders as normal text and not part of a two-column structure? The list is not following parallel construction. with this line tacked on to it.